docs/src/single-pool.mdx

```diff
@@ -0,0 +1,477 @@
+  'My Cool Pool',
+  'cPool',
+);
+
+// sign with the fee payer and authorized withdrawer
+```
+
+  </TabItem>
+</Tabs>
+
+A URL may also be included as an optional parameter.
+
+### Using a single-validator pool
+
+#### Depositing
+
+When a pool is created, its stake account is delegated to the appropriate vote account, and for that epoch, stake in an "activating" state can be deposited into it.
+After this epoch, stake must be in an "active" state to deposit into the pool.
+That is, it must be delegated to the vote account, and a deposit can only be performed after the next epoch boundary.
+
+Assuming the stake account `9cc4cmLcZA89fYmcVPPTLmHPQ5gab3R6jMqj124abkSi` is in an active state:
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool deposit 9cc4cmLcZA89fYmcVPPTLmHPQ5gab3R6jMqj124abkSi
+```
+
+When an explicit stake account address is provided, the CLI can determine the pool address automatically.
+
+  </TabItem>
+
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+```typescript
+const transaction = await SinglePoolProgram.deposit({
+  connection,
+  pool: poolAddress,
+  userWallet,
+  userStakeAccount,
+});
+
+// sign with the fee payer and stake account withdraw authority, if these signers differ
+// userWallet is a convenience parameter to use one account as a payer, authority, and lamport recipient
+```
+
+  </TabItem>
+
+  <TabItem value="jsn" label="WEB3.JS NEXT">
+
+```typescript
+const transaction = await SinglePoolProgram.deposit({
+  rpc,
+  pool: poolAddress,
+  userWallet,
+  userStakeAccount,
+});
+
+// sign with the fee payer and stake account withdraw authority, if these signers differ
+// userWallet is a convenience parameter to use one account as a payer, authority, and lamport recipient
+```
+
+  </TabItem>
+</Tabs>
+
+All versions of the `deposit` command/transaction automatically create the associated token account for the pool token if it doesn't exist and no auxiliary token account address is provided.
+
+The program also makes available a convenience address for each pool, called the default deposit address.
+This allows a flow where you create and delegate a stake at a program-derived address, and then can deposit this stake after the epoch boundary, without having to generate or keep track of any new keypairs.
+The user retains full authority on the stake account until they decide to deposit.
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool create-default-stake --pool DkE6XFGbqSyYzRugLVSmmB42F9BQZ7mZU837e2Cti7kb 1000000000
+```
+
+(some time passes)
+
+```console
+spl-single-pool deposit --pool DkE6XFGbqSyYzRugLVSmmB42F9BQZ7mZU837e2Cti7kb --default-stake-account
+```
+
+  </TabItem>
+
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+```typescript
+const minimumDelegation = (await connection.getStakeMinimumDelegation()).value;
```

Comment:
nit-up-to-you: how about putting a comment that any value greater than or equal to the minimum delegation is valid?

Reply:
you know, it occurs to me that until the minimum for stake program is activated, they can actually do this with any amount. the only enforcement done anywhere is that the _pool_ doesnt go below max(1sol, minimum). maybe to allay confusion ill just hardcode the number...